src/hugr/rewrite/replace.rs

```diff
@@ -114,30 +116,41 @@ impl Rewrite {
         )
     }
 
+    pub fn verify_convexity(&self) -> Result<(), ReplaceError> {
+        todo!()
+    }
+
+    pub fn verify_boundaries(&self) -> Result<(), ReplaceError> {
+        todo!()
+    }
+}
```

Comment:
```suggestion
    pub fn verify_convexity(&self) -> Result<(), ReplaceError> {
        unimplemented!()
    }

    pub fn verify_boundaries(&self) -> Result<(), ReplaceError> {
        unimplemented!()
    }
}
```

Reply:
I feel I want to leave this code untouched as much as possible given it is likely destined to the bin, but otoh - the plan is probably not "to do" these, so ok :-)